legal_db/views.py

```diff
@@ -1,6 +1,135 @@
-from django.shortcuts import render  # noqa
-from django.http import HttpResponse
+from legal_db.models import FAQ, Scholarship
+from taggit.models import Tag
+from django.shortcuts import render
+from django.urls import reverse
 
 
 def index(request):
-    return HttpResponse("Hello, world. This is the CC Legal Database project index.")
+    return render(request, "legal_db/index.html")
+
+
+def case_index(request):
+    context = {
+        "cases": [
```

Comment:
I don't think we should commit this data here, we should use a fixture to load test data into the database. 

https://docs.djangoproject.com/en/3.0/howto/initial-data/

Reply:
Same with other data

Reply:
Yes, I agree. This is just mocked data to show something in the template, this will not be in the final code. I was looking to change this PR to draft, is still in progress but I don't find the option in the new GitHub UI, I'll try changing to old interface.